screensaver.turnoff/addon.xml

```diff
@@ -1,13 +1,44 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="screensaver.turnoff" name="Turn Off" version="0.9.1" provider-name="Dag Wieers">
+<addon id="screensaver.turnoff" name="Turn Off" version="0.10.1" provider-name="dagwieers">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+  </requires>
+  <extension point="xbmc.ui.screensaver" library="screensaver.py"/>
   <extension point="xbmc.addon.metadata">
     <platform>all</platform>
-    <summary lang="en_GB">Screensaver that turns your screen off to save power</summary>
-    <description lang="en_GB">The Turn Off screensaver turns your TV, projector or monitor off, like any old fashioned screensaver is intended to do.</description>
-    <source>http://github.com/dagwieers/plugin.screensaver.off</source>
-    <license>GPL2+</license>
+    <summary lang="en_GB">Screensaver that turns your display off to save power</summary>
+    <description lang="en_GB">
+The Turn Off screensaver turns your TV, projector or screen off, like any old fashioned screensaver is intended to do.
+
+Next to managing your display, it can also manage your device, log your profile off or mute audio to avoid sounds through your A/V receiver.
+    </description>
+    <license>GNU General Public License, v2+</license>
     <email>dag@wieers.com</email>
+    <website>https://kodi.wiki/view/Add-on:Turn_Off</website>
+    <source>https://github.com/dagwieers/screensaver.turnoff</source>
+    <forum>https://forum.kodi.tv/showthread.php?tid=331076</forum>
+    <news>
+v0.10.1 (2019-10-30)
```

Comment:
Please keep this to a minimum and only for the last version.

Reply:
The other versions were never released. So this contains the condensed changelog since the previous release.

Reply:
I removed the v0.9.1 part as that version was already released before.

Although I am in favor to keep a condensed changelog going back a few versions, so users can relate changes (in behaviour) to a (recent) update.